Update fuzziness in fast/ for accelerated drawing

nmoucht opened the pull request:
#### 22276a499a287dd1e40ba4e583aa0e9b20cbd197
<pre>
Update fuzziness in fast/ for accelerated drawing
<a href="https://bugs.webkit.org/show_bug.cgi?id=246837">https://bugs.webkit.org/show_bug.cgi?id=246837</a>
rdar://101449704

Reviewed by Simon Fraser.

Update fuzziness in fast/ for when accelerated drawing is enabled by default.

* LayoutTests/fast/backgrounds/background-opaque-clipped-gradients.html:
* LayoutTests/fast/backgrounds/generated-bck-image-with-small-relative-size.html:
* LayoutTests/fast/backgrounds/hidpi-background-image-contain-cover-scale-needs-more-precision.html:
* LayoutTests/fast/backgrounds/scaled-gradient-background.html:
* LayoutTests/fast/borders/border-image-fill-inline-no-border.html:
* LayoutTests/fast/borders/border-image-repeat-stretch.html:
* LayoutTests/fast/borders/border-painting-correctness-dashed.html:
* LayoutTests/fast/borders/border-painting-correctness-dotted.html:
* LayoutTests/fast/borders/hidpi-double-border-with-border-radius-always-produce-solid-line.html:
* LayoutTests/fast/borders/outline-border-radius-simple.html:
* LayoutTests/fast/box-shadow/box-shadow-with-zero-radius.html:
* LayoutTests/fast/box-shadow/inset-box-shadow.html:
* LayoutTests/fast/css/text-decorations-on-first-line-and-containing-block.html:
* LayoutTests/fast/filter-image/clipped-filter.html:
* LayoutTests/fast/forms/date/date-editable-components/date-editable-components-placeholder-color.html:
* LayoutTests/fast/gradients/conic-center-outside-box.html:
* LayoutTests/fast/gradients/radial-two-hints.html:
* LayoutTests/fast/images/async-image-background-image-repeated.html:
* LayoutTests/fast/images/exif-orientation-background-image-repeat.html:
* LayoutTests/fast/images/hidpi-image-position-on-device-pixels-with-border-radius.html:
* LayoutTests/fast/images/image-map-outline-with-paint-root-offset.html:
* LayoutTests/fast/images/sprite-sheet-image-draw.html:
* LayoutTests/fast/layers/parent-clipping-overflow-is-overwritten-by-child-clipping.html:
* LayoutTests/fast/layers/wrong-clipping-semi-transparent-compositing-layer-on-subpixel-position.html:
* LayoutTests/fast/multicol/simple-line-layout-line-index-after-strut.html:
* LayoutTests/fast/repaint/gradients-em-stops-repaint.html:
* LayoutTests/fast/table/table-border-radius.html:
* LayoutTests/fast/text/empty-shadow.html:
* LayoutTests/fast/text/frequent-text.html:
* LayoutTests/fast/text/initial-advance-selected-text.html:
* LayoutTests/fast/transforms/skew-x-135deg-with-gradient.html:
* LayoutTests/fast/transforms/skew-y-135deg-with-gradient.html:

Canonical link: <a href="https://commits.webkit.org/255974@main">https://commits.webkit.org/255974@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/196eee65392258bd899026b8f2e22ad32c6a2327

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/94217 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/3409 "Built successfully") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/24974 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/103899 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/164171 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/3436 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/31623 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/86571 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/99907 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/99873 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/76/builds/3436 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/80629 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/86571 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/76/builds/3436 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/24974 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/86571 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/38005 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/24974 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/35889 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/24974 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/39767 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/80629 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1943 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/41710 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/24974 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
<!--EWS-Status-Bubble-End-->

smfr commented on LayoutTests/fast/borders/border-painting-correctness-dashed.html:
```diff
@@ -1,6 +1,7 @@
 <DOCTYPE html>
 <html>
 <head>
+<meta name="fuzzy" content="maxDifference=0-355; totalPixels=0-4808" />
```
255 is the largest difference.

nmoucht replied:
Fixed.

smfr commented on LayoutTests/fast/borders/border-painting-correctness-dotted.html:
```diff
@@ -1,6 +1,7 @@
 <DOCTYPE html>
 <html>
 <head>
+<meta name="fuzzy" content="maxDifference=0-355; totalPixels=0-15225" />
```
255 is the largest difference.

nmoucht replied:
Fixed.

smfr commented on LayoutTests/fast/text/frequent-text.html:
```diff
@@ -5,6 +5,7 @@
 </head>
 <body>
 <div id="target">Hello 👋 <span id="inside"></span> World!</div>
+<meta name="fuzzy" content="maxDifference=0-70; totalPixels=0-382" />
```
This should go in the `<head>`

nmoucht replied:
Fixed.

nmoucht committed:
Update fuzziness in fast/ for accelerated drawing
https://bugs.webkit.org/show_bug.cgi?id=246837
rdar://101449704

Reviewed by Simon Fraser.

Update fuzziness in fast/ for when accelerated drawing is enabled by default.

* LayoutTests/fast/backgrounds/background-opaque-clipped-gradients.html:
* LayoutTests/fast/backgrounds/generated-bck-image-with-small-relative-size.html:
* LayoutTests/fast/backgrounds/hidpi-background-image-contain-cover-scale-needs-more-precision.html:
* LayoutTests/fast/backgrounds/scaled-gradient-background.html:
* LayoutTests/fast/borders/border-image-fill-inline-no-border.html:
* LayoutTests/fast/borders/border-image-repeat-stretch.html:
* LayoutTests/fast/borders/border-painting-correctness-dashed.html:
* LayoutTests/fast/borders/border-painting-correctness-dotted.html:
* LayoutTests/fast/borders/hidpi-double-border-with-border-radius-always-produce-solid-line.html:
* LayoutTests/fast/borders/outline-border-radius-simple.html:
* LayoutTests/fast/box-shadow/box-shadow-with-zero-radius.html:
* LayoutTests/fast/box-shadow/inset-box-shadow.html:
* LayoutTests/fast/css/text-decorations-on-first-line-and-containing-block.html:
* LayoutTests/fast/filter-image/clipped-filter.html:
* LayoutTests/fast/forms/date/date-editable-components/date-editable-components-placeholder-color.html:
* LayoutTests/fast/gradients/conic-center-outside-box.html:
* LayoutTests/fast/gradients/radial-two-hints.html:
* LayoutTests/fast/images/async-image-background-image-repeated.html:
* LayoutTests/fast/images/exif-orientation-background-image-repeat.html:
* LayoutTests/fast/images/hidpi-image-position-on-device-pixels-with-border-radius.html:
* LayoutTests/fast/images/image-map-outline-with-paint-root-offset.html:
* LayoutTests/fast/images/sprite-sheet-image-draw.html:
* LayoutTests/fast/layers/parent-clipping-overflow-is-overwritten-by-child-clipping.html:
* LayoutTests/fast/layers/wrong-clipping-semi-transparent-compositing-layer-on-subpixel-position.html:
* LayoutTests/fast/multicol/simple-line-layout-line-index-after-strut.html:
* LayoutTests/fast/repaint/gradients-em-stops-repaint.html:
* LayoutTests/fast/table/table-border-radius.html:
* LayoutTests/fast/text/empty-shadow.html:
* LayoutTests/fast/text/frequent-text.html:
* LayoutTests/fast/text/initial-advance-selected-text.html:
* LayoutTests/fast/transforms/skew-x-135deg-with-gradient.html:
* LayoutTests/fast/transforms/skew-y-135deg-with-gradient.html:

Canonical link: https://commits.webkit.org/255974@main
